dialog/lib/_dialog.scss

```diff
@@ -192,7 +194,7 @@
   // Transitions for open/closed states
   .container {
     will-change: transform, opacity;
-    transition-property: transform;
+    transition-property: transform, opacity;
```

Comment:
I checked the spec, and opacity's timing function should be `linear`. Can you add that below?

```scss
transition-timing-function: var(--_closing-transition-easing), linear;
```

Reply:
Hm, it also has a kind of annoying content fade out before container fade out situation...

Do you mind adding a comment above `transition-property`?

```scss
// TODO: make opacity fade out content and container separately
```